src/Engines/thinEngine.ts

```diff
@@ -171,6 +171,29 @@ export class ThinEngine {
         Effect.ShadersRepository = value;
     }
 
+    /**
+    * Gets or sets the textures that the engine should not attempt to load as compressed
+    */
+    protected excludedCompressedTextures: string[] = [];

```

Comment:
needs to start with _ as it is a protected member _excludedCompressedTextures or the build will fail but other than that all looks good !!!!